resources/variants/ultimaker3_bb0.8.inst.cfg

```diff
@@ -24,9 +24,6 @@ infill_pattern = ='zigzag' if infill_sparse_density > 80 else 'triangles'
 infill_wipe_dist = 0
 jerk_enabled = True
```

Comment:
we could bring the jerk_enabled and the raft_jerk also to ultimaker.def.json, right?

Reply:
Then I would need to disable it for the UM2 machines, which is a larger list of machines. (The fdmprinter default for enable jerk is false). Therefore I chose to enable jerk in UM3, S3 and S5 explicitly.

Reply:
then bring it to the machine json (at least it should not be in the variant profiles, and it is in some quality profiles as well)

Reply:
Hmm, I thought I removed them all.... I will clean this up as well.